pr_agent/git_providers/gitlab_provider.py

```diff
@@ -87,8 +87,9 @@ def get_files(self):
         return [change['new_path'] for change in self.mr.changes()['changes']]
 
     def publish_description(self, pr_title: str, pr_body: str):
-        logging.exception("Not implemented yet")
-        pass
+        self.mr.title = pr_title
+        self.mr.description = pr_body
+        self.mr.save()
```

Comment:
**Suggestion:** Consider handling potential exceptions when setting the title and description of the merge request. If the setting fails, it would be good to have a fallback or at least log the error.
```suggestion
        try:
            self.mr.title = pr_title
            self.mr.description = pr_body
            self.mr.save()
        except Exception as e:
            logging.error(f'Failed to update MR title and description: {e}')
```